RELEASE_NOTES.md

```diff
@@ -6,10 +6,21 @@
 
 ## Upgrading
 
+- The `frequenz-client-microgrid` dependency was bumped to v0.4.0. If you are using the client directly in your code too you will need to upgrade too.
```

Comment:
I'd avoid the double `too` in the last sentence
```suggestion
- The `frequenz-client-microgrid` dependency was bumped to v0.4.0. If you are using the client directly in your code, you will need to upgrade too.
```

Reply:
Updated, thanks!